runtime/lua/vim/uri.lua

```diff
@@ -83,7 +89,12 @@ end
 
 -- Return or create a buffer for a uri.
 local function uri_to_bufnr(uri)
-  return vim.fn.bufadd((uri_to_fname(uri)))
+  local scheme = uri:match("^([a-z]+)://.*") or 'file'
```

Comment:
According to RFC, scheme must be included.
So I think if scheme is not included, raise error.
https://tools.ietf.org/html/rfc3986#section-3.1